src/css/style.css

```diff
@@ -0,0 +1,383 @@
+#homePage{
+    position:relative;
+    height: 7428px;
+}
+#header {
+    display:flex;
+    position: relative;
+    top:53px;
+    left:209px;
+    width:1502px;
+    height:65px;
+    
+}
+
+.logo{
+    margin-right: 33%;
+    width:153px;
+    height:65px;
+}
+.homeLinks {
+    /*outline:3px solid black;*/
+    justify-content: space-between;
+    margin-top:12px ;
+    margin-right: 20%;
```

Comment:
use a single margin property for top and right, i.e margin 10 10 10 10;

Reply:
Aright. Thank you.